lib/plug/adapters/cowboy/conn.ex

```diff
@@ -37,11 +37,7 @@ defmodule Plug.Adapters.Cowboy.Conn do
   def send_file(req, status, headers, path, offset, length) do
     %File.Stat{type: :regular, size: size} = File.stat!(path)
 
-    length =
-      cond do
-        length == :all -> size
-        is_integer(length) -> length
-      end
+    length = if length == :all, do: size, else: length
```

Comment:
@josevalim Oops!, you are right! Although, in the case of error, I think it's better to be explicit and say that we're waiting for an integer or an `:all` atom instead of `CondClauseError`. wdyt?